cmd/githubrunnerscalesetlistener/config/config_test.go

```diff
@@ -0,0 +1,92 @@
+package config
```

Comment:
Do we need to cover the rest of the scenarios? (LogLevel, LogFormat, Metrics, Certificate etc..)